webapp/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM debian:jessie
+ENV REFRESHED_AT 2015-05-18
+
+# update packages and prepare to build software
+RUN ["apt-get", "update"]
+RUN ["apt-get", "-y", "install", "build-essential"]
+
+ENV LANG en_US.utf8
+
+RUN ["apt-get", "-y", "install", "git"]
```

Comment:
Needed for installing through pip, when using a repository. Should probably be removed, since we don't use any repositories right now.